handout/testing/intro-to-tdd/setup/karma-config.md

```diff
@@ -0,0 +1,82 @@
+# Karma Configuration
+Karma is the foundation of our testing workflow, it brings together our other testing tools to define what framework we want to use, what environment to test under, what specific actions we want to perform, etc. In order to do this Karma relies on a configration file *karma.config.js*. You can seed a new configuration file though the `karma init` command, which will guide you through a few basic questions to get a bare minimum setup running. If we take a look at the *karma.config.js* file in ng2-redux-starter we'll see a few important points of interest.
+
+``` typescript
+module.exports = function(config) {
+	config.set({
+      	frameworks: [
```

Comment:
You had.  I am 99% sure I saw it in a PR somewhere.  I rebased off master before making this, but I don't know.  Anyhow a fix is on its way.